plugins/woocommerce/includes/class-woocommerce.php

```diff
@@ -1341,4 +1342,25 @@ public function register_remote_log_handler( $handlers ) {
 		$handlers[] = wc_get_container()->get( RemoteLogger::class );
 		return $handlers;
 	}
+
+	/**
+	 * Tracks the history WooCommerce Allow Tracking option.
+	 * - When the field was first set to allow tracking
+	 * - Last time the option was changed
+	 *
+	 * @param string $old_value The old value for the woocommerce_allow_tracking option.
+	 * @param string $value The current value for the woocommerce_allow_tracking option.
+	 * @since x.x.x
+	 *
+	 * @return void
+	 */
+	public function get_tracking_history( $old_value, $value ) {
```

Comment:
Looking at the contents of this function I only see it calling two "update" functions. It's also hooked into an "update" hook. Shouldn't the function name reflect the same thing, that it's updating the tracking history?